Xamarin.Forms.Platform.iOS/Cells/CellRenderer.cs

```diff
@@ -55,22 +56,27 @@ public virtual void SetBackgroundColor(UITableViewCell tableViewCell, Cell cell,
 
 		protected void UpdateBackground(UITableViewCell tableViewCell, Cell cell)
 		{
+			var uiBgColor = UIColor.White; // Must be set to a solid color or blending issues will occur
+
 			if (cell.GetIsGroupHeader<ItemsView<Cell>, Cell>())
 			{
-				if (UIDevice.CurrentDevice.CheckSystemVersion(7, 0))
-					SetBackgroundColor(tableViewCell, cell, new UIColor(247f / 255f, 247f / 255f, 247f / 255f, 1));
+				if (!UIDevice.CurrentDevice.CheckSystemVersion(7, 0))
+					return;
+
+				uiBgColor = new UIColor(247f / 255f, 247f / 255f, 247f / 255f, 1);
 			}
 			else
 			{
-				// Must be set to a solid color or blending issues will occur
-				var bgColor = UIColor.White;
-
 				var element = cell.RealParent as VisualElement;
-				if (element != null)
-					bgColor = element.BackgroundColor == Color.Default ? bgColor : element.BackgroundColor.ToUIColor();
-
-				SetBackgroundColor(tableViewCell, cell, bgColor);
+				if (element != null && element.BackgroundColor != Color.Default)
+					uiBgColor = element.BackgroundColor.ToUIColor();
 			}
+
+			var defaultBgColor = cell.OnThisPlatform().DefaultBackgroundColor();
+			if (defaultBgColor != Color.Default)
+				uiBgColor = defaultBgColor.ToUIColor();
```

Comment:
to me, it looks like, for non-groupheaders cells, the Default takes the priority to BgColor, which seems wrong

Reply:
@StephaneDelcroix yes, but i guess that now we have wrong behavior.
Let's imagine, that we want to have ListView with such backColor
```csharp
ListView.BackgroundColor = Color.Black.MultiplyAlpha(.5);
```
But also we want to have transparent cells (maybe, we have image is under the listView or something else)

But your approach doesn't provide such possibility, because backColor of Cell would be **Color.Black.MultiplyAlpha(.5)** as well.
So, in total we will see 0.5 black over 0.5 black (= 0.75 black)

We are loosing opportunity to set real transparent color for cell.

Reply:
@StephaneDelcroix don't you agree?

Reply:
@StephaneDelcroix currently non-header cells take BackgroundColor from parent's BackgroundColor (if it isn't **Default**). Maybe we are able apply the same flow for header cells as well, but it would be **breaking** changes.

But from my point of view it looks like rather old bug. All (Or most of all) Xamarin Views have transparent (Default) background color by default, but header cell - not.